### ISSUE TITLE
Bump to http ^1.0.0 to avoid conflicts with other packages

### ISSUE THREAD

**Issue by biklas7** [Iteo/connecteo#30]
The http package was [updated to v1](https://pub.dev/packages/http/changelog) some time ago and most packages started using that version. Can connecteo update to use the latest version as well? So we can avoid things like:
```
Because connecteo >=2.0.0 depends on http ^0.13.6 and package depends on http ^1.1.0, connecteo >=2.0.0 is forbidden.
So, because pacakge depends on connecteo ^2.0.0, version solving failed.
```

**Comment by Devi88** [comment]
Hello, we decided to release version 2.0.0 with http 0.13 because http 1.0.0 requires Dart 3.0 and Flutter version 3.10.
But today we released version 2.1.0 with updated dependencies.

**Comment by biklas7** [comment]
Thanks for the quick update! Yes, it works. Also thanks for this amazing package, it deserves more recognition 👍
